cucumber/kubernetes/features/authenticate_with_annotations.feature

```diff
@@ -0,0 +1,27 @@
+Feature: An authorized client can authenticate as a permitted role with the host's
```

Comment:
It isn't clear to me what is "permitted role" and therefore not clear what we are testing here. Do you mean that the client has necessary Conjur permissions to authenticate and application identity is defined in its annotations?

Reply:
fixed